build/common.project.props

```diff
@@ -47,6 +47,7 @@
     <TestUtilitiesDirectory>$(RepositoryRootDirectory)test\TestUtilities\</TestUtilitiesDirectory>
     <NuGetBuildLocalizationRepository>$(RepositoryRootDirectory)submodules\NuGet.Build.Localization\</NuGetBuildLocalizationRepository>
     <LocalizationRootDirectory>$(NuGetBuildLocalizationRepository)localize</LocalizationRootDirectory>
+    <LocalizationRepoRootDirectory>$(RepositoryRootDirectory)localize</LocalizationRepoRootDirectory>
     <LocalizationFilesDirectory>$(ArtifactsDirectory)LocalizedFiles</LocalizationFilesDirectory>
```

Comment:
These names are so similar, it's difficult to understand why they all exist, or how they're different. Additionally, looking at all the references to `LocalizationRootDirectory`, it appears you replaced them all with `LocalizationRepoRootDirectory`, although I wasn't careful in my check, so I could be wrong. But at a glance it appears we don't need the old property at all with this change.

Reply:
Some explanation:

- `LocalizationRootDirectory`: Submodule repo. Points to `.\submodules\NuGet.Build.Localization\localize\`
- `LocalizationRepoRootDirectory`: Simulated localize folder in NuGet.Client repo root. Points to `.\localize\`

Both folders have a copy of `NuGet.Build.Localization\localize\` repo content.

We still need `LocalizationRootDirectory` because some VSIX packing targets still refers to the submodule repo.

The copy I did for localization step (i.e., when you run `msbuild .\build\loc.proj ...`) only applies for that

Reply:
Discussed offline. Applied feedback on property names.